config/environments/production.rb

```diff
@@ -34,14 +36,4 @@
   end
 
   config.active_record.dump_schema_after_migration = false
-
-  config.paperclip_defaults = {
```

Comment:
this seems unrelated?

Reply:
It was dead code so I wanted to purge it.